### PR TITLE
Add page for new serverless view setup & troubleshooting

### PR DESCRIPTION
**Pull request by Hesperide**
Related to `dogweb` PR (https://github.com/DataDog/dogweb/pull/64123) making a change to https://docs.datadoghq.com/integrations/amazon_xray/ to add additional setup steps.

### CODE REVIEW

File: content/en/serverless/troubleshooting/connect_invoking_resources.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- your functions are written in Node.js or Python Lambda runtimes
+- [APM with Datadog's tracing libraries][2] (`dd-trace`) is configured on your functions
+- [APM with Datadog's X-Ray integration][2] is configured on your functions, and you've [configured Datadog's Lambda Library to enrich AWS X-Ray segments][4]
+- the managed resource invoking your function is one of the following: API Gateway, SQS, SNS, EventBridge, Kinesis, DynamoDB, S3.
+- your functions are instrumented with a recent version of Datadog's Lambda Library (>= `v3.46.0` for Node, >= `v28` for Python)
+
+[1]: /logs/explorer/saved_views
+[2]: /serverless/distributed_tracing#choose-your-tracing-library
+[3]: /serverless/installation
+[4]: /integrations/amazon_xray/#enriching-xray-segments-with-datadog-libraries
```
Comment (Hesperide) [inline content/en/serverless/troubleshooting/connect_invoking_resources.md:26]: We'll need to make sure this is valid/tweak this after merging https://github.com/DataDog/dogweb/pull/64123 and checking the URL.

File: content/en/serverless/troubleshooting/connect_invoking_resources.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Deeper Visibility into Resources Invoking Lambda Functions
+kind: documentation
+---
+
+{{< img src="serverless/serverless-view.png" alt="Serverless View" >}}
+
+By default, the Serverless view groups your serverless resources by service to help you easily visualize how each part of your application is performing. For each service, you will see the functions that belong to it, along with the resources (Amazon API Gateway, SNS, SQS, DynamoDB, S3, EventBridge, Kinesis) that invoked them. 
+
+While grouping by service is the default, you can also group your resources by AWS CloudFormation stack name, as well as any other tags you've configured (e.g., team, project, or environment). Additionally, [Saved Views](https://docs.datadoghq.com/logs/explorer/saved_views/) allows you to preserve your preferred way of grouping, so you don't need to manually enter it every time you visit the page.
```
Comment (Hesperide) [inline content/en/serverless/troubleshooting/connect_invoking_resources.md:10]:
```suggestion
While grouping by service is the default, you can also group your resources by AWS CloudFormation stack name, as well as any other tags you've configured (such as team, project, or environment).
```

File: content/en/serverless/troubleshooting/connect_invoking_resources.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Deeper Visibility into Resources Invoking Lambda Functions
+kind: documentation
+---
+
+{{< img src="serverless/serverless-view.png" alt="Serverless View" >}}
+
+By default, the Serverless view groups your serverless resources by service to help you easily visualize how each part of your application is performing. For each service, you will see the functions that belong to it, along with the resources (Amazon API Gateway, SNS, SQS, DynamoDB, S3, EventBridge, Kinesis) that invoked them.
```
Comment (Hesperide) [inline content/en/serverless/troubleshooting/connect_invoking_resources.md:8]:
```suggestion
By default, the Serverless view groups your serverless resources by service to help you easily visualize how each part of your application is performing. For each service, you see the functions that belong to it, along with the resources (Amazon API Gateway, SNS, SQS, DynamoDB, S3, EventBridge, Kinesis) that invoked them. 
```

File: content/en/serverless/troubleshooting/connect_invoking_resources.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+{{< img src="serverless/serverless-view.png" alt="Serverless View" >}}
+
+By default, the Serverless view groups your serverless resources by service to help you easily visualize how each part of your application is performing. For each service, you will see the functions that belong to it, along with the resources (Amazon API Gateway, SNS, SQS, DynamoDB, S3, EventBridge, Kinesis) that invoked them. 
+
+While grouping by service is the default, you can also group your resources by AWS CloudFormation stack name, as well as any other tags you've configured (e.g., team, project, or environment). Additionally, [Saved Views](https://docs.datadoghq.com/logs/explorer/saved_views/) allows you to preserve your preferred way of grouping, so you don't need to manually enter it every time you visit the page. 
+
+## Setup
+
+To instrument your Lambda functions for the first time, follow our [serverless installation instructions][3].
```
Comment (Hesperide) [inline content/en/serverless/troubleshooting/connect_invoking_resources.md:14]:
```suggestion
To instrument your Lambda functions for the first time, follow the [serverless installation instructions][3].
```

File: config/_default/menus/menus.en.yaml [hunk 5]
```diff
@@ -1057,10 +1057,14 @@ main:
     url: serverless/troubleshooting/serverless_tagging
     parent: serverless_troubleshooting
     weight: 702
+  - name: Connect Invoking Serverless Resources
```
Comment (Hesperide) [inline config/_default/menus/menus.en.yaml:1060]:
```suggestion
  - name: Visualizing Managed Resources
```

**Review by DylanLovesCoffee — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**